public/js/components/Editor.css

```diff
@@ -27,23 +27,28 @@
   width: 60px;
   height: 12px;
   position: absolute;
-  top: 0;
+  top: 1px;
   right: -4px;
 }
 
 .editor.new-breakpoint.breakpoint-disabled svg {
   opacity: 0.3;
 }
 
+.search-bar ~ .editor-mount {
+  height: calc(100% - 58px)
+}
+
 .editor-mount,
 .CodeMirror {
   width: 100%;
-  height: 100%;
+  height: calc(100% - 17px);
```

Comment:
unfortunately, the editor was being masked by the footer. I'd like to investigate avoiding absolute positioning here. I was able to avoid it with the search bar so it *should* be possible.